transform/operations_test.go

```diff
@@ -586,6 +586,75 @@ func TestConvertToInt64(t *testing.T) {
 	runOpTests(t, func() transformOperation { return &convertToInt64{} }, tests)
 }
 
+func TestConvertToBool(t *testing.T) {
+	tests := []opTests{
+		{
+			description: "Working string case",
+			in:          "1",
+			want:        true,
+		},
+		{
+			description: "Failing string case",
```

Comment:
Technically this is a "working" case as it properly transforms, so probably better to name it as such.

Reply:
You're right "working" is a poor choice of word, I'll replace Working/Failing with True/False

Reply:
https://github.com/GannettDigital/jstransform/pull/144/commits/5359590e5db0445ebe5d34025a18f4beeec28321